example/example_functions.go

```diff
@@ -1,17 +1,26 @@
 package example
 
-import "github.com/dailymotion/allure-go"
+import (
+	"github.com/dailymotion/allure-go"
+)
 
 func doSomething() {
-	allure.Step("Something", func() {
+	allure.Step(allure.Description("Something"), allure.Body(func() {
 		doSomethingNested()
-	})
+	}))
 }
 
 func doSomethingNested() {
-	allure.Step("Because we can!", func() {})
+	allure.Step(allure.Description("Because we can!"), allure.Body(func() {}))
 }
 
 func addSomeParameters(parameters map[string]interface{}) {
```

Comment:
It looks like this is not used. What is the purpose of this method?

Reply:
This was meant to illustrate a programmatic way of specifying parameters, but I removed a call to this function from the example test scripts.